<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndexGenerator</class>
 <widget class="QDialog" name="IndexGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dominant Colors â€“ Index Generator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="folderPathEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Path/to/ImageFolder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="folderSelector">
       <property name="text">
        <string>Select Image Folder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="indexPathEdit">
       <property name="placeholderText">
        <string>Path/to/save/ColorIndex</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="indexSelector">
       <property name="text">
        <string>Select Index Location</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="recursiveCheckBox">
     <property name="text">
      <string>Search folders recursevly</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="chancelButton">
       <property name="text">
        <string>Chancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="makeIndexButton">
       <property name="text">
        <string>Build Index</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="continueButton">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Set + Continue</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
